install/operator/pkg/apis/syndesis/v1beta1/syndesis_types.go

```diff
@@ -116,7 +116,7 @@ type DatabaseConfiguration struct {
 	Name string `json:"name,omitempty"`
 
 	// Host and port of the PostgreSQL database to access
-	URL string `url:"url,omitempty"`
+	URL string `json:"url,omitempty"`
```

Comment:
@phantomjinx do you think this change can be problematic? I think it was a typo

Reply:
Is this also a typo problem in v1alpha1?

Reply:
We don't have it in `v1alpha1`